readthedocs/doc_builder/environments.py

```diff
@@ -910,7 +911,8 @@ def get_container_name(self):
                 project_name=self.project.slug,
             )
         else:
-            name = f'sync-project-{self.project.pk}-{self.project.slug}'
+            uuid = uuid.uuid4()
+            name = f'sync-{uuid}-project-{self.project.pk}-{self.project.slug}'
```

Comment:
I'd put it at the end, it is the least important bit of data.

Reply:
however, it will be likely to be clipped there, which could lead to more non-deterministic bugs with syncing/building. most of the information in the container id is just for debugging and is infromational -- if we need a sha/uuid for functionality, it seems like the most important piece of information in the container id